apps/configurator/lib/configurator_web/components/layouts/app.html.heex

```diff
@@ -1,6 +1,6 @@
 <header class="px-4 sm:px-6 lg:px-8"></header>
 <main class="px-4 py-20 sm:px-6 lg:px-8">
-  <div class="mx-auto max-w-2xl">
+  <div class="mx-auto max-w-full">
```

Comment:
Did this due to the size of the tables and them not getting centered. If you have a better way please do tell, CSS is not my strong suit

Reply:
I have removed this styles in #722 so we can address it in another PR